PWGCF/DataModel/FemtoDerived.h

```diff
@@ -210,7 +210,8 @@ enum CharmHadronMassHypo {
   lcToPKPi = 1,
   lcToPiKP = 2
 };
-
+DECLARE_SOA_COLUMN(GIndexCol, gIndexCol, int);                      //! Global index for the collision
```

Comment:
Why is this a normal column and how do you plan to use it, specifically outside or inside O2Physics?

Reply:
yes, I use it outside for mixing events locally. I will merge the DF directly, so I also add timestamp